feature-libs/cart/base/occ/adapters/converters/occ-cart-normalizer.ts

```diff
@@ -41,6 +42,12 @@ export class OccCartNormalizer implements Converter<Occ.Cart, Cart> {
     return target;
   }
 
+  protected handleQuote(source: Occ.Cart, target: Cart) {
```

Comment:
I would suggest to rename this method to `handleQuoteCode()` because only quote code will be handled

Reply:
@ChristophHi, what do you think?